src/index.html

```diff
@@ -10,6 +10,10 @@
 
 <body>
   <h1>MMO Games</h1>
+  <footer>
+    Created by Tekle & Alick under "Liscense to be included later"
```

Comment:
<p> tag the whole text and the text in quotation whithin a <q> tag.

Reply:
Thanks for the input........working on it now